flink-rpc/flink-rpc-akka/pom.xml

```diff
@@ -29,10 +29,16 @@ under the License.
 		<relativePath>..</relativePath>
 	</parent>
 
-	<artifactId>flink-rpc-akka_${scala.binary.version}</artifactId>
+	<artifactId>flink-rpc-akka</artifactId>
 	<name>Flink : RPC : Akka</name>
 	<packaging>jar</packaging>
 
+	<properties>
+		<akka.version>2.6.15</akka.version>
+		<akka.scala.binary.version>2.12</akka.scala.binary.version>
+		<akka.scala.version>2.12.7</akka.scala.version>
```

Comment:
This is the same Scala 2.12 version that we use in the 2.12 Flink builds, just to reduce some complexity.